tests/test_examples.py

```diff
@@ -427,7 +427,7 @@ def test_krasnoselskii_mann_increasing_step_sizes(self):
         self.assertAlmostEqual(wc, ref_pesto_bound, delta=self.relative_precision * ref_pesto_bound)
 
     def test_wc_optimal_contractive_halpern_iteration(self):
-        n, gamma = 7, 2.15
+        n, gamma = 3, .36
```

Comment:
gamma should be >= 1 in OC-Halpern (at least in the paper it is)

Reply:
woops :) thanks!